xAPI.md

```diff
@@ -2020,6 +2020,21 @@ contain attachments.
 * The Client MAY send batches of type "application/json" where every attachment
 Object has a fileUrl, ignoring all requirements based on the "multipart/mixed" format.
 
+###### File URL
+The File URL is intended to provide a location from which the LRS or another system can retrieve the
+attachment. There are, however, no requirements for the owner of the attachment to make the 
+attachment data available at the location indefinately or to make the attachment publically
+available without security restrictions. When determining attachment hosting arrangements, 
+designers of systems that will send Statements using the "fileUrl" property are encouraged to 
+consider the needs of end recipient(s) of the Statement especially if the attachment content 
+is not included with the Statement.
+
+* The attachment data SHOULD be retirevable at the IRL indicated by the fileUrl.
```

Comment:
minor typo in "retirevable". Propose calling it a URL (which is commonly used for IRIs that represent URLs) instead of IRL (one day all those bits will be gone...). Fine with the text above.